pkg/controller/image-stream-tag-reconciler/reconciler.go

```diff
@@ -116,6 +133,9 @@ func (r *reconciler) Reconcile(req controllerruntime.Request) (controllerruntime
 }
 
 func (r *reconciler) reconcile(req controllerruntime.Request, log *logrus.Entry) error {
```

Comment:
Lots of debugging messages here -- do we want to keep those in? If so, maybe publish metrics instead?